assets/javascripts/job_details.js

```diff
@@ -0,0 +1,207 @@
+function create_element(tag, content=[], attrs={}) {
+	let elem = document.createElement(tag);
+
+	for (let key in attrs) {
```

Comment:
I know about this syntax but avoid it because it doesn't work in older browsers.

Reply:
We're already using it but admittedly only because I just started using it. The compatibility doesn't look too bad: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#Browser_compatibility